sample/package.json

```diff
@@ -1,40 +1,39 @@
 {
-  "name": "ng19-test",
+  "name": "ng20-test",
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "firebase emulators:exec --import seed \"ng serve\"",
+    "start": "npx --yes firebase-tools@latest emulators:exec --import seed \"ng serve\"",
```

Comment:
maybe someone had firebase-tools installed globally, but I don't and I'd prefer to follow the pattern I see in the root `package.json` of installing this on demand

alternatively, we could install the package in the `devDependencies` and keep it up to date